### ISSUE TITLE
Can't login to Nextcloud with v1.4.4

### ISSUE THREAD

**Issue by Sbroekie** [dgmid/nextcloud-notes-mac-client#55]
Nextcloud-notes-mac-client v1.4.3 is working just fine, but v1.4.4 does not show the login popup window to login to Nextcloud.

Version used:
MacBook Pro 2021 (M1)
Monterey 12.3.1 (but same issue with older versions of Monterey)

Popup window below does not appear with version v1.4.4:
![Screenshot 2022-05-19 at 08 49 54](https://user-images.githubusercontent.com/17739902/169229628-bf194655-2a5e-4a08-b717-8e556e632857.png)

**Comment by emmjam** [comment]
You have to log in to the window shown in the bottom right in order to establish the trust and connect your account. 
Log in/out is available from the Nextcloud Notes Client menu, right at the very top of the screen

![image](https://user-images.githubusercontent.com/31128722/173837430-2e6db9a7-bf19-468a-aa60-cd08467c26f7.png)

**Comment by Sbroekie** [comment]
Yes, I understand I have to login to that window (bottom right in my supplied screenshot). The issue is that that window does NOT show (popup) when using nextcloud notes v1.4.4.
